Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "c-ward"
-version = "0.15.40"
```

Comment:
Here and in the other Cargo.toml files, it's not necessary to bump the version number; I use `cargo release` to do the releases, which will do the bump automatically.

Reply:
oh ok, i'll then revert the changes